src/app/tests/suites/certification/Test_TC_DRLK_2_12.yaml

```diff
@@ -0,0 +1,435 @@
+
+    - label:
+          "TH writes WrongCodeEntryLimit attribute value as 3 on the DUT and
+          Verify that the DUT sends Success response"
+      PICS: DRLK.S.F00 && DRLK.S.F01 && DRLK.S.A0030
+      command: "writeAttribute"
+      attribute: "WrongCodeEntryLimit"
+      arguments:
+          value: 3
+
+    - label:
+          "TH writes WrongCodeEntryLimit attribute value as 3 on the DUT and
+          verify DUT responds with UNSUPPORTED_WRITE"
+      PICS: DRLK.S.F00 && DRLK.S.F01 && ! DRLK.S.A0030
+      command: "writeAttribute"
+      attribute: "WrongCodeEntryLimit"
+      arguments:
+          value: 3
+      response:
+          error: UNSUPPORTED_WRITE
+
+    - label:
+          "TH writes UserCodeTemporaryDisableTime attribute value as 15 Seconds
+          on the DUT and Verify that the DUT sends Success response"
+      PICS: DRLK.S.F00 && DRLK.S.F01 && DRLK.S.A0031
+      command: "writeAttribute"
+      attribute: "UserCodeTemporaryDisableTime"
+      arguments:
+          value: 15
+
+    - label:
+          "TH writes UserCodeTemporaryDisableTime attribute value as 15 Seconds
+          on the DUT and Verify that the DUT sends Success response"
+      PICS: DRLK.S.F00 && DRLK.S.F01 && ! DRLK.S.A0031
+      command: "writeAttribute"
+      attribute: "UserCodeTemporaryDisableTime"
+      arguments:
+          value: 15
+      response:
+          error: UNSUPPORTED_WRITE
+
+    - label: "TH sends the Unbolt Door command to the DUT with invalid PINCode"
+      PICS: DRLK.S.C27.Rsp
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "1234568"
+      response:
+          error: FAILURE
+
+    - label: "TH sends the Unbolt Door command to the DUT with invalid PINCode"
+      PICS: DRLK.S.C27.Rsp
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "1234568"
+      response:
+          error: FAILURE
+
+    - label: "TH sends the Unbolt Door command to the DUT with invalid PINCode"
+      PICS: DRLK.S.C27.Rsp
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "1234568"
+      response:
+          error: FAILURE
+
+    - label: "TH sends the Unbolt Door command to the DUT with invalid PINCode"
+      PICS: DRLK.S.C27.Rsp
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "1234568"
+      response:
+          error: FAILURE
+
+    - label:
+          "TH reads the UserCodeTemporaryDisableTime attribute from the DUT and
+          check attribute is triggered"
+      PICS: DRLK.S.F00 && DRLK.S.F01 && DRLK.S.A0031
+      command: "readAttribute"
+      attribute: "UserCodeTemporaryDisableTime"
```

Comment:
Reading this attribute does not check what the test plan says to check.  Why is there a read here at all?